sensitive-banner-static/res/common-banner.js

```diff
@@ -254,26 +273,28 @@ function getAmount() {
 
 function addBannerSpace() {
 	var expandableBannerHeight = $( 'div#WMDE_Banner' ).height() + 44,
-		bannerDivElement = $( '#WMDE_Banner' );
-
-	switch ( 'vector' ) {
-		// switch ( skin ) { TODO fix when non-static
-		case 'vector':
-			bannerDivElement.css( 'top', 0 - expandableBannerHeight );
-			$( '#mw-panel' ).animate( { top: expandableBannerHeight + 160 }, 1000 );
-			$( '#mw-head' ).animate( { top: expandableBannerHeight }, 1000 );
-			$( '#mw-page-base' ).animate( { paddingTop: expandableBannerHeight }, 1000 );
-			break;
-		case 'monobook':
-			$( '#globalWrapper' ).css( 'position', 'relative' );
-			$( '#globalWrapper' ).css( 'top', expandableBannerHeight );
-			bannerDivElement.css( 'top', '-20px' );
-			bannerDivElement.css( 'background', 'none' );
-			break;
+			bannerDivElement = $( '#WMDE_Banner' );
+
+	if ( showBanner ) {
```

Comment:
Inverting this condition to something like `if (!showBanner ) { return; }` would not make it necessary to wrap this awful `switch` in the condition what would maybe make it a little bit easier to read. But this is very detail in this case, not insisting on changing this